examples/05_transformers.ipynb

```diff
@@ -0,0 +1,849 @@
+    "\n",
+    "The `Series.index` and `DataFrame.index` are used for representing the time series or sequence index. `sktime` supports pandas integer, period and timestamp indices.\n",
+    "\n",
+    "`Panel`-s are commonly represented as:\n",
+    "* a `pandas.DataFrame` in a specific format, defined by the `pd-multiindex` mtype. This has a 2-level index, for time points and instances\n",
+    "* a `list` of `pandas.DataFrame`, where all `pandas.DataFrame` are in the `Series` format. The different `list` elements are the different instances\n",
+    "\n",
+    "In either case, the \"time\" index must be a `sktime` compatible time index type, as for `Series`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sktime.datatypes import get_examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-04-10T16:07:05.849641Z",
+     "iopub.status.busy": "2021-04-10T16:07:05.849188Z",
+     "iopub.status.idle": "2021-04-10T16:07:06.049023Z",
+     "shell.execute_reply": "2021-04-10T16:07:06.049499Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# example of a univariate series\n",
+    "get_examples(\"pd.Series\", \"Series\")[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of a multivariate series\n",
+    "get_examples(\"pd.DataFrame\", \"Series\")[1]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of a panel with mtype pd-multiindex\n",
+    "get_examples(\"pd-multiindex\", \"Panel\")[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# example of the same panel with mtype df-list\n",
+    "get_examples(\"df-list\", \"Panel\")[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`sktime` supports more mtypes, see the \"datatypes and datasets\" tutorial for more details."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.2 General transformer signature - time series transformers<a class=\"anchor\" id=\"section_1_2\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Transformers for `Series` and `Panel` have the same high-level interface. Depending which data type they are more commonly used for, they are found either in the `transformations.series` or `transformations.panel` module. As said, this does not imply a separate interface.\n",
+    "\n",
+    "The most important interface points of transformers are:\n",
+    "\n",
+    "1. construction with parameters, this is as with any other `sktime` estimator\n",
+    "2. fitting the transformer, via `fit`\n",
```

Comment:
I would reverse the order of definitions here going for `method`: description of what it does, possibly a signature. Sth along the lines of:

- `fit` fitting of the transformer parameters, accepts `(X, y)` ...